project/dexter-monitor/public/view/createServerDialog.html

```diff
@@ -0,0 +1,41 @@
+<md-dialog layout-padding="" >
+	<md-dialog-content>
+		<div>
+			<h2>Create new Dexter Server</h2>
+			</br>
+			<md-input-container class="md-block" flex-gt-xs="">
```

Comment:
But it would mean that we can create dexter-server on another machine. We are not able to do this.